Deleted the "allow start date to be in the past" toggle

BesherKitaz opened the pull request:
## Issue Description

Fixes #1448 
Delete the toggle "Allow start date to be in the past" and allow the user to pick any date in the past or future, prompting a message if in the past. Also change the multiple offerings toggle to be next to "start date."


![Screenshot 2025-06-13 094504](https://github.com/user-attachments/assets/3cbe528d-ed45-42a9-bd5b-a787f37a7ac2)

## Changes

- Deleted the "allow start date to be in the past" toggle on "Create Event Page"
- Now the user can select dates in the past in the calendar, always, without needing to switch the toggle
- The page will show a message in case the selected start date and time is in the past or/and the end date is in the past
- Changed the "This is a series of events." toggle location in the page 


![Screenshot 2025-06-13 094538](https://github.com/user-attachments/assets/357dbafe-127b-4f66-9b01-560263775712)


## Testing

- Open the CELTS website as an admin or Student Staff and go to "Create Event"
- Under "Programs" Choose any of the list of programs
- Go to start date and pick a date in the past or today
- Pick the start time to be in the past, and try end time in the future or past
- try the program on Firefox since there was a problem there
- Try edge cases in time like 12pm, 12:30pm, etc...

PuskarC committed:
Removed toggle and then worked with the calendar

BesherKitaz committed:
Merge branch 'development' into startDate

BesherKitaz committed:
Added a warning message if the event is in the past or happening now. Fixed the center alignment of toggle

PuskarC committed:
Checking commit differences

PuskarC committed:
Removed a toggle and then fixed the functionality for Calendar in Firefox. Works for all browser now

PuskarC committed:
Finishing merge

BesherKitaz committed:
fixed toggle position

BesherKitaz committed:
Merge branch 'development' into startDate

BesherKitaz committed:
resolving merge conflict

Karina-Agliullova commented:
Your changes break the modal that we have for multiple offerings. Find what in your code is breaking it and fix it. I checked it in development, there is no such a problem.
![Screenshot (80)](https://github.com/user-attachments/assets/f9f04cf3-3a7a-41c6-a741-1967fff68441)

BesherKitaz committed:
removed an extra div that was causing problems, and fixed indentations in JS file

BesherKitaz commented:
We fixed the problem of the problem of the Modal of the Multiple offerings. There was an extra div in the HTML that was breaking it and we removed it. It should work now after the new pushes.

Karina-Agliullova commented:
I found another problem. When I click on the event that is happening right now, it displays the phrase that you added, which is good. But if I try to make a weekly event with a modal, and I set it up to be in the past and save it, the phrase doesn't disappear and still stays on the page, which makes it now a false statement. 
![Screenshot (83)](https://github.com/user-attachments/assets/2da21cd0-ec9a-403b-9ccd-1810bae733cb)

BesherKitaz committed:
Fixed the problem of the red prompt when multiple offerings date picker is used

BesherKitaz committed:
Fixed the problem of adding the text back when it is untoggled and the date is in the past

BesherKitaz commented:
I fixed the problem in the new push. I just made sure it the text is removed when the toggle is active (checked), and to add the text again when the toggle is unchecked.

Karina-Agliullova committed:
Merge branch 'development' into startDate

Karina-Agliullova commented:
Another issue. When I select an event for any date, it lets me use the calendar, which is nice. But when I click on the modal, it is breaking when I save a series of evens, and then uncheck the toggle for multiple series of events, as the calendar disappears and doesn't reappear after the toggle is unchecked. On development it is not an issue, it works correctly, so something in your code is breaking the calendar

BesherKitaz committed:
The toggle was disapearing when checked, which makes it impossibe to untoggle. This issue was fixed.

BesherKitaz committed:
Fixing git commits

BesherKitaz commented:
I fixed it now. Please let me know if there are any other issues.

Karina-Agliullova reviewed (approved):
Looks good! Great work!
